tools/ci/annotate_dm.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+
+set -euo pipefail
+python3 tools/ci/annotate_dm.py "$@"
```

Comment:
shouldn't we be using the vendored version of python that we have for this? look at how other scripts do it

Reply:
oh never mind, that's for `bat` stuff. my bad

Reply:
I used this as a reference

https://github.com/tgstation/tgstation/blob/773d80b53f1e3322b2544d59a548737c8da7cd24/tools/ci/check_changelogs.sh#L5

Reply:
> oh never mind, that's for `bat` stuff. my bad

both sides use it, so why not use it too